droplets_test.go

```diff
@@ -178,6 +178,110 @@ func TestDroplets_Destroy(t *testing.T) {
 	}
 }
 
+func TestDroplets_Kernels(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v2/droplets/12345/kernels", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Note that kernels is nullable on droplets, can you add a test covering that behavior?